packages/augur-ui/src/modules/reporting/common.tsx

```diff
@@ -1068,7 +1068,7 @@ export interface ParticipationTokensViewProps {
   purchasedParticipationTokens: FormattedNumber;
   tokensOwned: FormattedNumber;
   percentageOfTotalFees: FormattedNumber;
-  participationTokensClaimable: FormattedNumber;
+  pastParticipationTokensPurchased: FormattedNumber;
```

Comment:
I've changed the name because those are the tokens that the user bought (REP), not the ones he can claim (DAI). It was kinda confusing